<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDiagnosisWidget</class>
 <widget class="QWidget" name="MainDiagnosisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Список правил системы помощи в диагностике</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="m_view">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="m_addRule">
          <property name="toolTip">
           <string>Добавить правило</string>
          </property>
          <property name="type" stdset="0">
           <string>add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_editRule">
          <property name="toolTip">
           <string>Редактировать правило</string>
          </property>
          <property name="type" stdset="0">
           <string>edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_deleteRule">
          <property name="toolTip">
           <string>Удалить правило</string>
          </property>
          <property name="type" stdset="0">
           <string>remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
